### ISSUE TITLE
Feature request: Share a link that opens the Android app at a given location

### ISSUE THREAD

**Issue by sarahmaddox** [sarahmaddox/techcomm-map-android#9]
As a user, I'd like to share a link to a particular item on the map, so that the person I share it with can open the map directly at the specific location. The link should open the Android app at the given location.

**Comment by asheeshjanghu** [comment]
Does this mean when I share an event from our app with some other person, that person should be able to open that message in our techcomm-map app and take him to map in our app on the location of the event? Do we need anything else also other than taking user to the location (like some tag)?

**Comment by sarahmaddox** [comment]
That's right - the link should open the Android app at the given location of the event. We do currently have a "Share" option for each event. When the user taps "Share", they can choose whether to share via email/twitter/etc, using the Android share intent. The app then nicely wraps up the location into a URL, like this one:
https://goo.gl/yixpkU

But that URL opens the web app. There's no hook into the Android app, even if the user who receives the shared message is currently on Android.

See these docs on Android intent handling:
https://developer.android.com/training/app-links/index.html
https://developer.android.com/training/app-links/index.html#url-handling

Note that this isn't as trivial as it seems at first glance. Last time I tried it, I ran into issues because the web app is hosted on GitHub pages. There's therefore no easy way of registering the necessary JSON asset links file on the GitHub pages for the website. 

Last time I investigated this was in October 2016. It may be that things have changed since then, or that someone is able to find another way of doing this.
